websites/F/Freek/metadata.json

```diff
@@ -0,0 +1,56 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.12",
+	"apiVersion": 1,
+	"author": {
+		"id": "567885938160697377",
+		"name": "Anaxes"
+	},
+	"service": "Freek",
+	"description": {
+		"en": "Freek.to is a free movie, tv show and more streaming application, supporting high quality streams and friendly UI and UX experiences."
+	},
+	"url": "freek.to",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/wmHltUl.png",
+	"thumbnail": "https://i.imgur.com/YU9QbjM.png",
+	"color": "#121212",
+	"category": "videos",
+	"tags": [
+		"movies",
+		"anime",
+		"tv",
+		"manga"
+	],
+	"iframe": true,
+	"iFrameRegExp": ".*",
```

Comment:
Are you able to use a more narrow regex?

Reply:
I'm not really sure what it iframe regex is. Is it just limiting the domain sources of the iframe?

Reply:
Yes, right now `.*` (meaning anything), will make the iframe.js script load into all iframes, regardless of URL.

It would be nice if you could limit it to a certain domain or path for example, so it doesn't load into Google Analytics iframe for example.